ehitchhiking-ui/src/app/shared/components/registration/registration.component.sass

```diff
@@ -0,0 +1,101 @@
+fieldset
+  border: none
+  margin: 0
+  box-shadow:  0 0 0.5rem 0.1rem rgba(221, 221, 221, 1)
+
+input
+  border: none
+  font-family: inherit
+  font-size: inherit
+  outline: none
+
+a
+  color: #999
+  text-decoration: none
+
+a:hover
+ color: #698f3f
+
+.center
+  display: flex
+  justify-content: center
+
+#login-form
+  margin: auto
+  width: 40%
+  padding: 5%
+
+.page
```

Comment:
Your page should have relative position. Why do you want to change z-index ?

Reply:
remove z-index